<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conference Speaker Server</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralWidget {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,  stop: 0 #FFFFFF, stop: 1 #CCCCCC);
	font-family: Myriad Pro;
	font: 9pt;
	color: &quot;#2E2E2E&quot;;
}

 QTabBar::tab:selected {
     border-color: #9B9B9B;
     border-bottom-color: #C2C7CB; /* same as pane color */
 }

QWidget#centralWidget &gt; * {
	font-family: Myriad Pro;
	font: 9pt;
	color: #2E2E2E;

}

QWidget#clientsBox {
    background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(186, 186, 186, 255), stop:1 rgba(212, 212, 212, 255));
    border: 1px solid #2D2D2D;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>16</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Control</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="chatLabel">
          <property name="text">
           <string>Chat: (N)</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="channelBox"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Voting</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Sound</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Stats</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>About</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 0);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="clientsBox">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="horizontalSpacing">
        <number>4</number>
       </property>
       <property name="verticalSpacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Server name:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Server address:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLineEdit" name="labelName">
         <property name="minimumSize">
          <size>
           <width>126</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>126</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Conference name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>255.255.255.255</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QScrollArea" name="clientsArea">
         <property name="styleSheet">
          <string notr="true">gridline-color: rgb(170, 0, 127);</string>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="clientBox">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>219</width>
            <height>69</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="Line" name="line">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>15</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 0;
background-image: url(:/res/images/face.png);</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
